Assets/Scripts/Controllers/TradeController.cs

```diff
@@ -63,6 +64,8 @@ public void CallTradeShipTest(Furniture landingPad)
         SpriteRenderer spriteRenderer = go.AddComponent<SpriteRenderer>();
         spriteRenderer.sprite = SpriteManager.GetSprite("Trader", "BasicHaulShip");
         spriteRenderer.sortingLayerName = "TradeShip";
+        controller.Animation = new SpritenameAnimation("flying", new string[] { "BasicHaulShip_01", "BasicHaulShip_02" }, 1 / 4);
```

Comment:
@Raketfart you where right, I'll have to look at your code. :( But I might learn something.